.github/workflows/deploy_production.yml

```
@@ -0,0 +1,64 @@
+name: Deploy Production
+
+on:
+  push:
+    tags:
+      - production-release
+  workflow_dispatch:
+
+jobs:
+  build_and_push_image:
+    name: Build Production Image
+    runs-on: ubuntu-latest
+    steps:
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v1
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+
```

I think you want to use the buildx system in docker (the new one)
```suggestion
    - name: Set up Docker Buildx
       uses: docker/setup-buildx-action@v2

```

---

I've started a build on #2531 with BuildX enabled. If that works, I'll add it here too.